test/integration/monitoring-alert/monitoring_alert_test.go

```diff
@@ -0,0 +1,42 @@
+package monitoring_alert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMonitoringAlertSecret(t *testing.T) {
+	secretT := tft.NewTFBlueprintTest(t)
+
+	secretT.DefineVerify(func(assert *assert.Assertions) {
+		secretT.DefaultVerify(assert)
+
+		projectDescribe := gcloud.Runf(t, "projects describe %s", secretT.GetStringOutput("project_id"))
```

Comment:
How about store `secretT.GetStringOutput("project_id") ` into a variable since we are using multiple times?